xtask/src/lib.rs

```diff
@@ -282,6 +285,19 @@ impl Package {
         format!("{self}-v{version}")
     }
 
+    pub fn is_version_acceptable(&self, version: &semver::Version) -> bool {
```

Comment:
Imo we should just take the bump amount and and unconditionally change it to `Patch` when `package == esp-rom-sys`, I think a good place to do this would be in `check_crate_before_bumping`. I think it'll get annoying if we have to error out and manually change this crate everytime.

Reply:
Can we just set the current version to `0.1.0` here now? I wasn't sure if the tooling needs to see a bump in order to include a crate in the release plan (but probably it works 🤔 )